### PR TITLE
LRQA-69709 Workflow Automated Test - CanSeeStateBeforeAssign

### PR DESCRIPTION
**Pull request by alanafergs**
https://issues.liferay.com/browse/LRQA-69709

### CODE REVIEW

**Comment by alanafergs** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase [hunk 1]
```diff
@@ -62,11 +62,46 @@ definition {
...
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Workflow.gotoAssetViaTableTitle(workflowAssetTitle = "userfn userln");
 
-		// TODO LRQA-69709 CanSeeStateBeforeAssign pending implementation
+		AssertElementPresent(
+			key_state = "Review",
+			locator1 = "AssetWorkflow#ASSET_STATE");
```
Comment (andreefarias) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase]: Look for a macro for that, maybe Workflow.viewTaskAssignedToMyRoles make this. Thanks!

**Review by andreefarias — CHANGES_REQUESTED** [review]

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase [hunk 2]
```diff
@@ -62,11 +62,46 @@ definition {
 	}
 
 	@description = "This is a test for LRQA-69709. Verify that the user can see the actual state of the user in the workflow process before assigning to itself. "
-	@ignore = "Test Stub"
 	@priority = "3"
```
Comment (andreefarias) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase:65]: Change priority to "4". Thanks!

**Review by andreefarias — CHANGES_REQUESTED** [review]

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase (hunk 2 above)
Comment (andreefarias) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase:65]: Change priority to "4". Thanks!

**Review by andreefarias — CHANGES_REQUESTED** [review]

**Pull request closed without merging**